config/manager/kustomization.yaml

```diff
@@ -19,6 +22,7 @@ configMapGenerator:
   - default-config/db-service.yaml
   - default-config/db-service-hl.yaml
   - default-config/db-secret.yaml
-  - default-config/backend-auth-configmap.yaml
-  - default-config/dynamic-plugins-configmap.yaml
+  - default-config/dynamic-plugins.yaml
+  - default-config/app-config.yaml
```

Comment:
nit: can you sort this list alphabetically? I think it would help the reviewers.

Reply:
+1